app/Http/Controllers/DashboardController.php

```diff
@@ -82,7 +79,7 @@ public function getIndex()
             }
 
 
-            return View::make('dashboard')->with('asset_stats', $asset_stats)->with('recent_activity', $recent_activity);
```

Comment:
Yeah, that line is removed.  I think the way github is displaying the diff is confusing :)